letterboxd/src/main/java/com/ataulm/letterboxd/LetterboxdComponent.kt

```diff
@@ -0,0 +1,24 @@
+package com.ataulm.letterboxd
+
+import android.app.Application
+import dagger.BindsInstance
+import dagger.Component
+
+@LetterboxdScope
+@Component(
+    modules = [
+        LetterboxdModule::class
+    ]
+)
+interface LetterboxdComponent {
+
+    val letterboxdRepository: LetterboxdRepository
```

Comment:
This is the only dependency we want to expose from this component. This is the API of the `letterboxd` (Gradle) module.